.github/workflows/no-response.yml

```
@@ -7,20 +7,32 @@ on:
     # Schedule for five minutes after the hour, every hour
     - cron: "5 * * * *"
 
+permissions:
+  issues: write
+  pull-requests: write
+
 jobs:
   noResponse:
     if: github.repository == 'remix-run/remix'
     runs-on: ubuntu-latest
 
     steps:
       - name: 🥺 Handle Ghosting
-        uses: lee-dohm/no-response@v0.5.0
+        uses: actions/stale@v5.0.0
```

You can just use latest version by doing

```suggestion
        uses: actions/stale@v5
```

---

👍 copy pasta 🤪